packages/core/src/model.ts

```diff
@@ -22,11 +22,9 @@ export function createModel<
   TComputedEvent = 'events' extends keyof TFinalModelCreators
     ? UnionFromCreatorsReturnTypes<TFinalModelCreators['events']>
     : never,
-  TComputedAction = 'actions' extends keyof TModelCreators
-    ? 'actions' extends keyof TFinalModelCreators
-      ? UnionFromCreatorsReturnTypes<TFinalModelCreators['actions']>
-      : any
```

Comment:
Using `any` should be almost forbidden in our types at the "output" positions. It has a domino effect on types and the inference - any unions containing that etc are becoming `any` themselves with that.